<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarHopperDialog</class>
 <widget class="QDialog" name="StarHopperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Star-Hopper Results</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="DetailsButton">
      <property name="text">
       <string>Details</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GotoButton">
      <property name="text">
       <string>Center</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="directionsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>directionsLabel</zorder>
  <zorder>horizontalWidget</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
